scripts/spack/packages/mfem/mfem-4.7.patch

```diff
@@ -7,6 +7,6 @@ index 00d50283b..c9be1ddd0 100644
 +# All changes in mfem-4.7.patch are already in MFEM since 4.7.0.1.
 +# Original patch: https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/mfem/mfem-4.7.patch
```

Comment:
Can this patch be removed now? Was this patch incompatible with 4.7.0.2?

Reply:
Sadly no. Spack is working on a new functionality called `drop` that will allow us not to have to override existing patches with noop files.